packages/app-bridge/src/AppBridgePlatformApp.spec.ts

```diff
@@ -0,0 +1,64 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { afterEach, beforeAll, describe, expect, it, vi } from 'vitest';
+import { notify } from './utilities';
+import { AppBridgePlatformApp } from './AppBridgePlatformApp';
+import { InitializationError } from './errors';
+
+const TOKEN = 'AjY34F87Dsat^J';
+
+describe('AppBridgePlatformApp', () => {
+    beforeAll(() => {
+        vi.mock('./utilities/subscribe', () => ({
+            subscribe: vi.fn().mockResolvedValue({ test: 'passed' }),
+        }));
+
+        vi.mock('./utilities/notify', () => ({
+            notify: vi.fn(),
+        }));
+    });
+
+    afterEach(() => {
+        vi.clearAllMocks();
+    });
+
+    it('should be instantiable', () => {
+        expect(new AppBridgePlatformApp()).toBeInstanceOf(AppBridgePlatformApp);
+    });
+
+    it('should not initialize without a token', async () => {
+        window.location.search = '?token=';
+
+        const platformApp = new AppBridgePlatformApp();
+
+        await expect(() => platformApp.dispatch({ name: 'openConnection' })).rejects.toThrow(new InitializationError());
```

Comment:
```suggestion
        await expect(() => platformApp.dispatch(openConnection())).rejects.toThrow(new InitializationError());
```

Reply:
looks nice and i like the function that we can pass, but until now i didnt see this pattern (at least not with dispatch) where do we want to put these dispatch command methods?

Reply:
`packages/app-bridge/src/registries/commands`